game/exceptions.py

```diff
@@ -34,3 +34,18 @@ class NotOwnerException(APIException):
 class GameSessionFullException(APIException):
     status_code = 400
     default_detail = {'Game Session': ['Game session is full.']}
+
+
+class BuildingNameException(APIException):
+    status_code = 400
+    default_detail = {'Building': ['Building name is invalid.']}
```

Comment:
You can leave it as it is or change it to BuildingNotFoundException with a 404 code.